examples/jsm/loaders/FBXLoader.js

```diff
@@ -427,31 +427,20 @@ class FBXTreeParser {
 
 		let texture;
 
-		const extension = textureNode.FileName.slice( - 3 ).toLowerCase();
+		const nonNativeExtensions = new Set( [ 'tga', 'tif', 'tiff', 'exr', 'dds', 'hdr', 'ktx2' ] );
 
-		if ( extension === 'tga' ) {
+		const extension = textureNode.FileName.split( '.' ).pop().toLowerCase();
 
-			const loader = this.manager.getHandler( '.tga' );
+		if ( nonNativeExtensions.has( extension ) ) {
 
-			if ( loader === null ) {
-
-				console.warn( 'FBXLoader: TGA loader not found, creating placeholder texture for', textureNode.RelativeFilename );
-				texture = new Texture();
-
-			} else {
-
-				loader.setPath( this.textureLoader.path );
-				texture = loader.load( fileName );
-
-			}
-
-		} else if ( extension === 'dds' ) {
-
-			const loader = this.manager.getHandler( '.dds' );
+			const loader = this.manager.getHandler( `.${extension}` );
 
 			if ( loader === null ) {
 
-				console.warn( 'FBXLoader: DDS loader not found, creating placeholder texture for', textureNode.RelativeFilename );
+				console.warn(
+					`FBXLoader: ${extension.toUpperCase()} loader not found, creating placeholder texture for`,
+					textureNode.RelativeFilename
+				);
 				texture = new Texture();
 
 			} else {
```

Comment:
With the new code, if a `fbx` file uses `psd` as textures, we no longer log a warning...? 🤔

Reply:
Correct.
I searched globally for `psd` and did not find another place where it is handled specifically.
I was thinking: do we need a dedicated handling for things that are not supported?
I mean, it does not need to be just `.psd`.
It could be a hardcoded black list (possibly quite large) of things that we should throw a warning for if encountered.
That's why a dropped that branch.

Reply:
Ideally we could just do this:

```js
const loader = this.manager.getHandler( `.${extension}` );

if ( loader !== null ) {

  texture = loader.load( fileName );

} else {

  console.warn(
	  `FBXLoader: ${extension.toUpperCase()} loader not found, creating placeholder texture for`,
	  textureNode.RelativeFilename
  );
  texture = new Texture();

}
```

But I guess for that to work we may need to add default handlers in `LoadingManager`?

https://github.com/mrdoob/three.js/blob/dev/src/loaders/LoadingManager.js#L140C1-L140C66

```js
const DefaultTextureLoader = /*@__PURE__*/ new TextureLoader();

const DefaultLoadingManager = /*@__PURE__*/ new LoadingManager();
DefaultLoadingManager.setHandler( /\.png$/i, DefaultTextureLoader );
DefaultLoadingManager.setHandler( /\.jpeg$/i, DefaultTextureLoader );
DefaultLoadingManager.setHandler( /\.jpg$/i, DefaultTextureLoader );
DefaultLoadingManager.setHandler( /\.gif$/i, DefaultTextureLoader );
```

Or even populate [this array](https://github.com/mrdoob/three.js/blob/dev/src/loaders/LoadingManager.js#L11) directly?

@Mugen87 What do you think?

Reply:
Um, I'm not sure I understand why it is required to add default handlers. 

If a loader for an extension isn't found, an empty texture is returned. 

I was never a fan of the following branch in `FBXLoader`:
```js
else {

    // TextureLoader#load() returns a texture in any case which can be used as placeholder if the image failed to load.
    texture = this.textureLoader.load( fileName );

}
```
Simply doing `texture = new Texture();` is better, imo.

Reply:
To rephrase: I vote for https://github.com/mrdoob/three.js/pull/28515#discussion_r1618370300 but without adding default handlers.

Reply:
> Simply doing `texture = new Texture();` is better, imo.
`const loader = this.manager.getHandler( `.${extension}` );` will return null for native images which are supposed to be handled without regex map. That's why there is this fallback : 
`texture = this.textureLoader.load( fileName );`

Reply:
Couldn't we have a second set with native extensions intended for `TextureLoader` (or `ImageBitmapLoader`). Meaning `[ 'png', 'jpeg', ' jpg', 'gif' ]. If an extension is not in that set, we check the non-native one ( `[ 'tga', 'tif', 'tiff', 'exr', 'dds', 'hdr', 'ktx2' ]`). If the check is `false`, we return an empty texture.

Reply:
I see the will to simplify the logic especially those nested `if` branches. I pushed a new commit. The only reason for keeping around non native extensions is to be able to throw a warning in the console. If we won't want that warning, we don't need them around making the `loader = specificLoader || defaultTextureLoader` simplifying even more.
I belive though that we want that warning there, hence keeping around non native extensions set.